src/Pages/Home/Home.jsx

```diff
@@ -10,11 +10,16 @@ const Home = () => {
     <main className={style.home}>
       <h1>Home Page</h1>
       <Button type="button" />
-      {articleRespState.loading && <Loading />}
-      {articleRespState.error && <div>Error</div>}
-      {articleRespState.response.map((item, index) => {
-        return <ArticleCard data={item} id={index} key={item?.id} />;
-      })}
+      <div className={style.articlesection}>
+        <div className={style.article}>
+          {articleRespState.loading && <Loading />}
+          {articleRespState.error && <div>Error</div>}
+          {articleRespState.response.map((item, index) => {
+            return <ArticleCard data={item} id={index} key={item?.id} />;
```

Comment:
For a single line return from an arrow function, [you don't need to write return](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#syntax)